src/main/java/io/pivotal/cfapp/task/AppDetailReadyDecider.java

```diff
@@ -0,0 +1,46 @@
+package io.pivotal.cfapp.task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import io.pivotal.cfapp.config.PivnetSettings;
+import io.pivotal.cfapp.domain.Space;
+
+@Component
+public class AppDetailReadyDecider {
```

Comment:
I think this should be renamed to be `AppDetailReadyToBeCollectedDecider` to pair with `AppDetailReadToBeRetrievedEvent`.  We are in fact triggering collection only when the decider makes a decision.